nemo/export/quantize/quantizer.py

```diff
@@ -212,38 +188,24 @@ def _sample_output(model):
             },
         )
 
-        if torch.distributed.get_rank() == 0:
-            print(f'Example NeMo output after PTQ: {response["sentences"]}"')
-
-    def quantize(
-        self,
-        model_file: str,
-        dataloader: Optional[List[List[str]]],
-        tensor_model_parallel_size: Optional[int] = None,
-        pipeline_model_parallel_size: Optional[int] = None,
-    ):
-        """Quantize model checkpoint using given dataloader and optional custom parallelism settings."""
-        model = self._load_model(model_file, tensor_model_parallel_size, pipeline_model_parallel_size)
+        logging.info(f'Example NeMo output before export: {response["sentences"]}"')
 
-        if self.quantization_config.algorithm is None:
-            return model
+    def quantize(self, model: MegatronGPTModel, forward_loop: Callable[[MegatronGPTModel], None]):
+        """Quantize the model and calibrate using given forward loop."""
+        assert self.quant_cfg is not None, "Quantization algorithm is not set"
 
-        model.set_inference_config(OmegaConf.to_container(self.inference_config))
-
-        def forward_loop(model):
-            print("Calibrating the model...")
-            for i, batch in enumerate(tqdm(dataloader)):
-                model.predict_step(batch, i)
+        logging.info(f"Quantizing model to {self.quantization_config.algorithm}...")
+        Quantizer._setup(model)
```

Comment:
let's please write `Quantizer` -> `self` (I realize it's a static method, but it just looks odd; I want to make sure a user instantiates a `Quantizer`, OK?)